ReactiveSwift.podspec

```diff
@@ -18,4 +18,6 @@ Pod::Spec.new do |s|
   # Directory glob for all Swift files
   s.source_files  = "Sources/*.{swift}"
   s.dependency 'Result', '~> 3.1'
+
+  s.pod_target_xcconfig = {"OTHER_SWIFT_FLAGS[config=Release]" => "-suppress-warnings" }
```

Comment:
Warnings can be silenced by using `inhibit_all_warnings!` in `Podfile` at users side, so I'm not confident if this is really needed.